ConTabs/OutputBuilder.cs

```diff
@@ -61,7 +61,7 @@ private void NoDataLine()
             {
                 var noDataText = "no data";
                 int colWidths = table._colsShown.Sum(c => c.MaxWidth);
-                int innerWidth = colWidths + (3 * table._colsShown.Count) - 1;
+                int innerWidth = colWidths + (2 * table._colsShown.Count * table.Padding) + (table._colsShown.Count + 1) - 2;
```

Comment:
Could this be simplified?

```C#
(2 * table._colsShown.Count * table.Padding) + table._colsShown.Count - 1;
```

Reply:
I doubt it.

Reply:
My point is that `+ (table._colsShown.Count + 1) - 2` is aritmetically equivalent to `+ table._colsShown.Count -1`. The parentheses don't change anything and +1 - 2 = -1.

Reply:
oh. It was more intuitive to me so I left it like that. I will change that.